packages/cli/src/commands/plugin/rollup.config.ts

```diff
@@ -41,6 +44,9 @@ export default {
     postcss(),
     imageFiles(),
     json(),
-    typescript(),
+    typescript({
+      include: `${paths.resolveTarget('src')}/**/*.{js,jsx,ts,tsx}`,
+      exclude: [paths.resolveTarget('src/testUtils/**')],
```

Comment:
Should one exclude `.test.` and `.storybooks.`, or is that not relevant as long as they aren't imported?

Reply:
This is just a primitive safeguard agains people importing testUtils from non-test code. Thinking it's not needed for tests and storybooks, those really shouldn't be imported by mistake :grin: